public/js/util/DevToolsUtils.js

```diff
@@ -0,0 +1,13 @@
+function reportException(who, exception) {
+  let msg = who + " threw an exception: ";
+  console.error(msg, exception);
+}
```

Comment:
I've never loved `reportException` anyway, and I'm not sure we even need it. Most errors like this (usually used at the end of a promise chain) could be automatically caught in the promise middleware if we want to do something special with them.